index.html

```diff
@@ -218,91 +216,94 @@ <h1>See project</h1>
-                               some of my work and experience!
-                               If you like what you see and
-                               have a project you need coded, don’t hesitate to contact me.
-                           </p>
-                       </div>
-
-                       <div class="primary_text">
-                           <h3>LET’S CONNECT</h3>
-                           <ul class="icons">
-                               <li>
-                                <a href="#"><img src="icons/Vector (2).svg" alt="gitHub"></a>
-                               </li>
-                               <li>
-                                <a href="#"> <img src="icons/Vector.svg" alt="tweeter"></a>
-                               </li>
-                               <li>
-                                <a href="#"><img src="icons/Vector (1).svg" alt="m-company"></a>
-                               </li>
-                               <li>
-                                <a href="#"><img src="icons/Linkedin icon.svg" alt="Linkedin"></a>
-                               </li>
-                            </ul>
+                        <div class="about-section">
+                            <div class="portfolio-header">
+                                <h2>
+                                About Myself
+                                </h2>
+                           </div>
+                           <div class="portfolio-desc">
+                               <p class="text-paragraph">
+                                   I’m a software developer! I can help you build a product , feature or website Look through
+                                   some of my work and experience!
+                                   If you like what you see and
+                                   have a project you need coded, don’t hesitate to contact me.
+                               </p>
+                           </div>
+
+                           <div class="primary_text">
+                               <h3>LET’S CONNECT</h3>
+                               <ul class="icons">
+                                   <li>
+                                    <a href="#"><img src="icons/Vector (2).svg" alt="gitHub"></a>
+                                   </li>
+                                   <li>
+                                    <a href="#"> <img src="icons/Vector.svg" alt="tweeter"></a>
+                                   </li>
+                                   <li>
+                                    <a href="#"><img src="icons/Vector (1).svg" alt="m-company"></a>
+                                   </li>
+                                   <li>
+                                    <a href="#"><img src="icons/Linkedin icon.svg" alt="Linkedin"></a>
+                                   </li>
+                                </ul>
```

Comment:
- [ ] As stated in the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the alignment for the About section and language list just like the Figma Design.
- Your About section Design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/230193930-4210d89f-4225-4cdf-aaec-5f5d713663e2.PNG)

- The Figma Design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/230194602-46b86f7b-15e0-47e0-b0e5-1545c98c692c.PNG)